### PR TITLE
Add jupyter_server_fileid

### PR DESCRIPTION
**Pull request by fcollonval**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~~If static libraries are linked in, the license of the static library is packaged.~~
- ~~Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).~~
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Fixes https://github.com/jupyter-server/jupyter_server_fileid/issues/14

### CODE REVIEW

**Comment by fcollonval** [comment]
@davidbrochart @dlqqq would you like to be added as maintainer?

**Comment by davidbrochart** [comment]
Thanks for creating this conda-forge package!

File: recipes/jupyter-server-fileid/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  number: 0
+
+requirements:
+  host:
+    - hatchling >=1.0
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - jupyter_server >=1.15,<3
```
Comment (davidbrochart) [inline recipes/jupyter-server-fileid/meta.yaml:24]: I'm still confused, don't we need `jupyter_server >=2`?

Comment (fcollonval) [reply recipes/jupyter-server-fileid/meta.yaml:24]: I used the constrain of the published package.

Comment (dlqqq) [reply recipes/jupyter-server-fileid/meta.yaml:24]: File ID does not have a hard dependency on JS2, though it is currently needed to track in-band edits.

**Comment by dlqqq** [comment]
@fcollonval Thank you for tackling this! And yes, I'd be happy to serve as a maintainer.

**Comment by fcollonval** [comment]
@conda-forge/help-python this is ready for review

**Review by dopplershift — APPROVED** [review]

**Pull request merged**